js/components/TagModal/TagModal.js

```diff
@@ -0,0 +1,41 @@
+import React, { Component } from 'react';
+import { View, Text, TouchableOpacity, Modal } from 'react-native';
+import styles from './styles';
+import PropTypes from 'prop-types';
+
+const TagModal = ({ open, close, title, description, backgroundColor }) => {
```

Comment:
the background color is not being used. Please remove it.

Reply:
It will be used when implemented in the page that's why we kept it .